### PR TITLE
update to api-platform/core:^2.7-alpha.2

### PR DESCRIPTION
**Pull request by soyuka**

### CODE REVIEW

File: src/Infrastructure/Shared/ApiPlatform/Metadata/QueryOperation.php [hunk 1]
```diff
@@ -97,7 +101,7 @@ public function __construct(
         // remove the extra query parameter
         unset($args[1]);
 
-        parent::__construct(self::METHOD_GET, ...array_values($args));
```
[the remark this answers is not in the document]
Comment (mtarld) [reply src/Infrastructure/Shared/ApiPlatform/Metadata/QueryOperation.php:100]: Indeed, but we unset a value just above, with make a "key whole". Do you see?

[the remark this answers is not in the document]
Comment (chalasr) [reply src/Infrastructure/Shared/ApiPlatform/Metadata/QueryOperation.php:100]: still fine, integer keys are renumbered when using the unpacking operator

[the remark this answers is not in the document]
Comment (mtarld) [reply src/Infrastructure/Shared/ApiPlatform/Metadata/QueryOperation.php:100]: :+1:

**Comment by mtarld** [comment]
LGTM, thanks @soyuka :slightly_smiling_face: 

A follow-up PR will come, that will remove either `Query` and `Command` operations metadata as well as data transformers to use exclusively providers/processors

**Pull request merged**